management/user_test.go

```diff
@@ -132,4 +168,8 @@ func TestUser(t *testing.T) {
 			m.User.Delete(auth0.StringValue(user.ID))
 		}
 	})
+
+	m.Role.Delete(auth0.StringValue(r1.ID))
+	m.Role.Delete(auth0.StringValue(r2.ID))
```

Comment:
Perhaps move this close to the creation using `defer`? Just to keep the consistency with other tests.